locale/en_US/translations.ts

```diff
@@ -1404,5 +1404,65 @@
             <source>Skip</source>
             <translation>Skip</translation>
         </message>
+        <message>
+            <source>Sort by</source>
+            <translation>Sort by</translation>
+        </message>
+        <message>
+            <source>Search is unavailable because the API does not support searching inside boxsets.</source>
+            <translation>Search is unavailable because the API does not support searching inside boxsets.</translation>
+        </message>
+        <message>
+            <source>Search is unavailable because the API does not support searching inside My List.</source>
+            <translation>Search is unavailable because the API does not support searching inside My List.</translation>
+        </message>
+        <message>
+            <source>Search this library</source>
+            <translation>Search this library</translation>
+        </message>
+        <message>
+            <source>Mark As Unplayed</source>
+            <translation>Mark As Unplayed</translation>
+        </message>
+        <message>
+            <source>Mark As Played</source>
+            <translation>Mark As Played</translation>
+        </message>
+        <message>
+            <source>Go To Series</source>
```

Comment:
Go To Series already exists in the translation file, as does Go To Season. Double check your list here so we don't duplicate entries.